src/map/items/item.cpp

```diff
@@ -301,6 +301,17 @@ void CItem::setSignature(int8* signature)
     memcpy(m_extra + 0x0C, signature, sizeof(m_extra) - 0x0C);
 }
 
+/************************************************************************
+*                                                                       *
+*  Appraisal Origin IDs                                                 *
+*                                                                       *
+************************************************************************/
+
+void CItem::setAppraisalID(uint8* appID)
+{
+    memcpy(m_extra + 0x16, &appID, sizeof(m_extra) - 0x16);
```

Comment:
`ref<uint8>(m_extra, 0x16) = appID`

also uint8*? This doesn't need to be a pointer.